README.md

```diff
@@ -94,6 +94,15 @@ Lint only changed files, skip lint on start.
 
 Path to `stylelint` instance that will be used for linting.
 
+### `fix`
+
+- Type: `Boolean`
+- Default: `false`
+
+With --fix option stylelint will fix as many errors as possible. The fixes are made to the actual source files. All unfixed errors will be reported.
+
+[Learn more](https://stylelint.io/user-guide/cli#autofixing-errors)
```

Comment:
Use description from node-api https://stylelint.io/user-guide/node-api#fix

Reply:
Thanks, subtle difference but should be clearer this way